api/src/main/resources/applicationContext-service.xml

```diff
@@ -95,89 +95,19 @@
 	<!--  **************************  DATA ACCESS OBJECTS  *************************  -->
 
 	<bean id="contextDAO" class="org.openmrs.api.db.hibernate.HibernateContextDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-		<property name="userDAO" ref="userDAO"/>
+		<constructor-arg ref="sessionFactory"/>
 	</bean>
 
-	<bean id="patientDAO" class="org.openmrs.api.db.hibernate.HibernatePatientDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-		<property name="searchSessionFactory" ref="searchSessionFactory"/>
-	</bean>
-	<bean id="personDAO" class="org.openmrs.api.db.hibernate.HibernatePersonDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-		<property name="searchSessionFactory" ref="searchSessionFactory"/>
-	</bean>
-	<bean id="conceptDAO" class="org.openmrs.api.db.hibernate.HibernateConceptDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-		<property name="searchSessionFactory" ref="searchSessionFactory"/>
-	</bean>
-	<bean id="userDAO" class="org.openmrs.api.db.hibernate.HibernateUserDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="obsDAO" class="org.openmrs.api.db.hibernate.HibernateObsDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="encounterDAO" class="org.openmrs.api.db.hibernate.HibernateEncounterDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="locationDAO" class="org.openmrs.api.db.hibernate.HibernateLocationDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="orderDAO" class="org.openmrs.api.db.hibernate.HibernateOrderDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="orderSetDAO" class="org.openmrs.api.db.hibernate.HibernateOrderSetDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="formDAO" class="org.openmrs.api.db.hibernate.HibernateFormDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
 	<bean id="adminDAO" class="org.openmrs.api.db.hibernate.HibernateAdministrationDAO">
 		<property name="sessionFactory" ref="sessionFactory"/>
 	</bean>
-	<bean id="programWorkflowDAO" class="org.openmrs.api.db.hibernate.HibernateProgramWorkflowDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="templateDAO" class="org.openmrs.api.db.hibernate.HibernateTemplateDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
-	<bean id="cohortDAO" class="org.openmrs.api.db.hibernate.HibernateCohortDAO">
-		<property name="sessionFactory" ref="sessionFactory"/>
-	</bean>
 	<bean id="serializedObjectDAO" class="org.openmrs.api.db.hibernate.HibernateSerializedObjectDAO"
 		  factory-method="getInstance">
 		<property name="sessionFactory" ref="sessionFactory"/>
 		<property name="supportedTypes">
 			<list/>
 		</property>
 	</bean>
```

Comment:
Why didn't you convert it? You can delete getInstance method and make it a regular bean with public constructor.